cmd/test_load.go

```diff
@@ -240,14 +241,23 @@ func loadAndConfigureTest(
 	return test.consolidateDeriveAndValidateConfig(gs, cmd, cliConfigGetter)
 }
 
+func loadSystemCertPool(gs *globalState) {
```

Comment:
```suggestion
func loadSystemCertPool(gs *logrus.FieldLogger) {
```
We can keep the dependency smaller, it isn't required to pass the entire state.